runtime/autoload/man.vim

```diff
@@ -406,4 +419,19 @@ function! man#init_pager() abort
   endif
 endfunction
 
+function! man#goto_tag(pattern, flags, info) abort
+  " currently no support for section completion
+  let sect = ""
```

Comment:
I suppose here we could parse pattern for a section specifier, and if one isn't found, perhaps fallback to `b:man_sect`

Reply:
Decided to go with this approach, except without falling back to `b:man_sect` - almost all man pages have `man(X)` references, so we'd be unlikely to need to fallback to a section, so I think it would be unnecessary complexity/code for no benefit.